ddtrace/span.py

```diff
@@ -139,7 +139,7 @@ def __init__(
         self.trace_id = trace_id or _rand.rand64bits(check_pid=_check_pid)  # type: int
         self.span_id = span_id or _rand.rand64bits(check_pid=_check_pid)  # type: int
         self.parent_id = parent_id  # type: Optional[int]
-        self.tracer = tracer
+        self.tracer = tracer  # type: Optional[Tracer]
```

Comment:
this should already be this type from argument definition no?

Reply:
Yeah I don't think it's necessary but I'm fine if you want to add more than necessary.